respa_admin/templates/common/_period_day.html

```diff
@@ -0,0 +1,35 @@
+{% with forloop.counter0 as dayId %}
+    <div class="weekday-row input-row" id="period-day-{{ id }}-{{ dayId }}">
+
+        <div id="day-db-ids">
+            {{ day.period }}
+            {{ day.id }}
+        </div>
+
+        <span class="select-dropdown btn btn-primary">
+            {{ day.weekday }}
+            <span class="glyphicon glyphicon-chevron-down" aria-hidden="true"></span>
+        </span>
+
+{#        {{ day.instance.get_weekday_display }}#}
+
+        <div class="form-group">
+            <div class="input-row range-input">
+                {{ day.opens }}
+                <span class="glyphicon glyphicon-minus"></span>
+                {{ day.closes }}
+            </div>
+        </div>
+
+        {{ day.closed }}
+
+{#        <button class="btn copy-next" type="button">#}
+{#            <span class="glyphicons glyphicons-more-windows icon-left" aria-hidden="true"></span>#}
+{#            Kopioi seuravaan#}
+{#        </button>#}
+{##}
+{#        <button class="btn add-time left-group" type="button" id="btn-add-new-day">#}
+{#            <span class="glyphicon glyphicon-plus-sign icon-left" aria-hidden="true"></span>Lisää toinen aika#}
+{#        </button>#}
```

Comment:
Avoid having commented out a.k.a "dead code" in the PRs.

Reply:
Actually dead code means code that is uncommented but never called. Anyway, both are bad dead code and commented code.

Reply:
Hmm.. seems that I was wrong too. :smile: According to Wikipedia [dead code](https://en.wikipedia.org/wiki/Dead_code) means code that is executed, but whose results are never used. Correct term for code that is never called is actually [Unreachable code](https://en.wikipedia.org/wiki/Unreachable_code).